ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessActivityResource.scala

```diff
@@ -35,7 +36,7 @@ class ProcessActivityResource(processActivityRepository: ProcessActivityReposito
         canWrite(processId) {
           entity(as[Array[Byte]]) { commentBytes =>
             complete {
-              val comment = new String(commentBytes, java.nio.charset.Charset.forName("UTF-8"))
+              val comment = SystemComment(new String(commentBytes, java.nio.charset.Charset.forName("UTF-8")))
```

Comment:
In this place it is not a comment created by system (designer) but by user. Please by the way change `Charset.forName` to `StandardCharsets.UTF_8`